src/vsCodeConfig/settings.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ConfigurationTarget, Uri, workspace, WorkspaceConfiguration } from "vscode";
+import { extensionPrefix } from '../constants';
+
+/**
+ * Uses extensionPrefix 'appService' unless otherwise specified
+ */
+export async function updateGlobalSetting<T = string>(section: string, value: T, prefix: string = extensionPrefix): Promise<void> {
+    const projectConfiguration: WorkspaceConfiguration = workspace.getConfiguration(prefix);
+    await projectConfiguration.update(section, value, ConfigurationTarget.Global);
+}
+
+/**
+ * Uses extensionPrefix 'appService' unless otherwise specified
+ */
+export async function updateWorkspaceSetting<T = string>(section: string, value: T, fsPath: string, prefix: string = extensionPrefix): Promise<void> {
```

Comment:
Would be nice if this PR also had changes like this one:
https://github.com/microsoft/vscode-azureappservice/pull/1060/files#diff-caaef5e8ec24b167d4d47bee5dec061bL31

It's a very simple change - it's related to adding this file - and it happens in quite a few files

Reply:
Will do after advancedCreation gets in since a lot of workspace setting stuff deals with that.